tests/integration/test_messages_integration.py

```diff
@@ -925,6 +945,7 @@ def test_error_api_quietly(capsys, mode):
 
     expected_err = [
         Line("Invalid channel."),
+        Line("Detailed information: {'message': 'Invalid channel.', 'code': 'BAD-CHANNEL'}"),
         Line(f"Full execution log: {str(emit._log_filepath)!r}"),
     ]
     expected_log = [
```

Comment:
It's not letting me quote the whole block for some reason, so I can't just leave a code suggestion comment, but this change makes `expected_err` equal to `expected_log`, so they may as well be merged into one variable.